sim/paladin/forbearance.go

```diff
@@ -0,0 +1,33 @@
+package paladin
+
+import (
+	"github.com/wowsims/sod/sim/core"
+	"time"
+)
+
+func (paladin *Paladin) registerForbearance() {
+
+	actionID := core.ActionID{SpellID: 25771}
+
+	forbearanceAura := paladin.RegisterAura(core.Aura{
+		Label:    "Forbearance",
+		ActionID: actionID,
+		Duration: time.Minute * 1,
+	})
+
+	paladin.OnSpellRegistered(func(spell *core.Spell) {
+
+		if spell.Flags.Matches(SpellFlag_Forbearance) {
+			oldEffect := spell.ApplyEffects
+
+			spell.ApplyEffects = func(sim *core.Simulation, unit *core.Unit, spell *core.Spell) {
+				oldEffect(sim, unit, spell)
+				forbearanceAura.Activate(sim)
+			}
+
+			spell.ExtraCastCondition = func(sim *core.Simulation, target *core.Unit) bool {
+				return !forbearanceAura.IsActive()
+			}
```

Comment:
You may want to grab the old function as well just in case so that this doesn't accidentally obscure adding extra cast conditions in forbearance spells and them not working correctly